Google.PowerShell.IntegrationTests/GcloudCmdlets.ps1

```diff
@@ -59,3 +59,37 @@ function Reset-GCloudConfig($oldConfig, $configName) {
     gcloud config configurations activate $oldConfig.NAME 2>$null
     gcloud config configurations delete $configName -q 2>$null
 }
+
+# Install Cloud SDK non-interactively.
+function Install-CloudSdk() {
+    Invoke-WebRequest -Uri "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-129.0.0-windows-x86_64-bundled-python.zip" `
```

Comment:
Please tease out the Uri into a separate variable.

Also, we should have some logic to "find the latest" Cloud SDK. The URL you have google-cloud-sdk-129.0.0 is already out of date. The lastest release is 131.

I believe this URL is always going to point to the latest and greatest:
https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz